schema/validator.go

```diff
@@ -38,7 +41,7 @@ func (e ValidationError) Error() string {
 }
 
 // Validate validates the given reader against the schema of the wrapped media type.
-func (v Validator) Validate(src io.Reader) error {
+func (v Validator) Validate(src io.Reader, verbose bool, out *log.Logger) error {
```

Comment:
Is there way we can do this without just piling on new arguments? Usually, one can tell they are on the path to spaghetti code when orthogonal arguments show up throughout the call chain.

`Validate` should only validate.